### PR TITLE
ci: run the sdk-compatability test from the release branch against main

### PR DESCRIPTION
**Pull request by bmwill**

### CODE REVIEW

**Review by aching — APPROVED** [review]

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -657,6 +657,14 @@ jobs:
         env:
           JSON_RPC_URL: "http://127.0.0.1:8080"
           FAUCET_URL: "http://127.0.0.1:8000"
+      - uses: actions/checkout@v2
+        with:
+          ref: diem-core-v1.2.0
+      - name: run sdk-compatability tests from release
+        run: cd sdk/compatability && cargo test -- --include-ignored
```
Comment (aching) [inline .github/workflows/ci-test.yml:664]: Minor nit: compatibility

Comment (bmwill) [reply .github/workflows/ci-test.yml:664]: welp turns out i'm bad at spelling :D and this is the name of the directory in the release branch lol

**Review by sausagee — APPROVED** [review]

File: .github/workflows/ci-test.yml (hunk 1 above)
Comment (sausagee) [reply .github/workflows/ci-test.yml]: i kinda like compatability :p

**Pull request merged**